cirq-core/cirq/experiments/qubit_characterizations.py

```diff
@@ -529,16 +529,16 @@ def single_qubit_state_tomography(
     """
     circuit_z = circuit + circuits.Circuit(ops.measure(qubit, key='z'))
```

Comment:
I do think it would be good to check here whether the key `'z'` is being used in the given circuit, and if so pick a unique key. One thing that could still fail is if the key is already used in the circuit to measure two or more qubits (e.g. `cirq.measure(q0, q1, key='z')`) then adding another instance here with a single qubit will fail.